app/src/main/java/com/bashkevich/androidfundamentals/model/entity/MovieEntity.kt

```diff
@@ -0,0 +1,32 @@
+package com.bashkevich.androidfundamentals.model.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.bashkevich.androidfundamentals.model.dto.GenreDto
+import kotlinx.serialization.SerialName
+
+@Entity(tableName = "movies")
+class MovieEntity(
+    @PrimaryKey
+    @ColumnInfo(name = "movie_id")
+    val movieId: Int,
+    @ColumnInfo(name = "title")
+    val title: String,
+    @ColumnInfo(name = "poster_path")
+    val posterPath: String,
+    @ColumnInfo(name = "backdrop_path")
+    val backdropPath: String,
+    @ColumnInfo(name = "runtime")
+    val runtime: Int?,
+    @ColumnInfo(name = "genres")
+    val genres: String?,
```

Comment:
Why not many to many?

Reply:
I think it's unnecessary because of bigger impact on performance

Reply:
I will only trust the numbers :) The performance here is not so obvious.